packages/ui/src/Dashboard/DetailView/ActionsDetails/BuySell/BuySell.tsx

```diff
@@ -52,27 +56,23 @@ class BuySell extends Component<Props, State> {
       const response = await apiCall('/v1/order/place', 'POST', payload);
       const error = getErrorMessage(response);
       if (error) {
-        console.log('Error');
-        console.log(error);
-        console.log(payload);
-        console.log(response);
-      } else {
-        console.log('Placed');
-        console.log(payload);
-        console.log(response);
+        this.props.showMessagePopup('error', response.errors.join(' , '));
```

Comment:
error is of type string. You can display the variable error. If you want to display all errors, I would suggest you do the errors.join() in the getErrorMessage method in Util.ts and return the string which has all the errors.

Reply:
Since variable 'errors' is an array of all the errors, will this do?
```suggestion
        this.props.showMessagePopup('error', response.errors[0]);
```

Reply:
You can use the variable 'error' itself, which has the value returned by getErrorResponse

Reply:
Ok done👍